test/infrastructure/docker/main.go

```diff
@@ -102,7 +103,9 @@ func main() {
 
 	ctrl.SetLogger(klogr.New())
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	restConfig := ctrl.GetConfigOrDie()
+	restConfig.UserAgent = remote.DefaultClusterAPIUserAgent("cluster-api-docker-controller")
```

Comment:
`cluster-api-docker-controller` consistently with all the others?

Reply:
👍🏼. Addressed.